src/transformers/models/phi/modeling_phi.py

```diff
@@ -634,6 +635,7 @@ def forward(
         output_attentions: Optional[bool] = False,
         use_cache: Optional[bool] = False,
         past_key_value: Optional[Tuple[torch.Tensor]] = None,
+        **kwargs,
```

Comment:
not 100% sure why we need these? (Was in Llama because padding mask was deprecated at some point but useless here IMO)